EcpBtBot/DialogFlow/Config.py

```diff
@@ -0,0 +1,18 @@
+
+#"""
+#This file contains a dico of keys=subject values=tokens
+#TODO: This class will be changed by an external file that will contain
+#subjects and tokens. 
+#"""
+
+tokens = {"Hôtel":'e3b0442b092c4e38a4cad3d9441d384f',
```

Comment:
it would be better a have a configuration file elsewhere that contains this information, rather than hard coded. I would use either a json file, or a yaml file and pass the name of the file as argument of your objects. This way you could also change the bots without changing the code....